tests/console/check_system_info.pm

```diff
@@ -35,6 +39,23 @@ sub run {
         assert_script_run("grep -w $milestone_version /etc/issue");
```

Comment:
If this will be run for all migration test, the check for milestone need be excluded for non-milestone test.

Reply:
Only set setting, this code can work. If don't set MILESTONE_VERSION, this code will not work.

Reply:
It is better to update the condition to make sure this only for milestone test although without the setting there is no failure.

Reply:
I think this condition is enough, this setting is written when we trigger cases. Daily will not use it. And even daily build add this setting. it should not be failure.